db/migrate/20211126080118_add_index_to_deleted_at_to_deleted_dossiers.rb

```diff
@@ -0,0 +1,5 @@
+class AddIndexToDeletedAtToDeletedDossiers < ActiveRecord::Migration[6.1]
+  def change
+    add_index :deleted_dossiers, [:deleted_at]
```

Comment:
```suggestion
    add_concurrent_index :deleted_dossiers, [:deleted_at]
```

Ca permet de pas locker la table